Pages/Table.php

```diff
@@ -2936,7 +2936,7 @@ function print_field_value($field, &$row = null) {
         if (!empty($field['render_'.$this->action.'_field']) && is_callable($field['render_'.$this->action.'_field'])) {
             return $field['render_'.$this->action.'_field']($row);
         } elseif (!empty($field['display_value']) && is_callable($field['display_value'])) {
-            return $field['display_value']($row);
+            return $field['display_value']($row, $field['field']);
```

Comment:
this shouldn't be necessary since this function should be in the description for a field - so the field already knows what field it's in. for example it would look like this :
presets = {
    my_field => { (this is the name of the field passed in the second parameter you added)
        type => '',
        display_value => function(){
            return '';
        }
    } 
}